README.md

```diff
@@ -6,18 +6,21 @@ A web app where you can find contributors for your open-source project or as an
 Find people who have the same vision as you do and contribute to each other for building amazing products.
 
 ## Connect with us
+As a community, we always encourage people to share their thoughts and ideas. Do you want to talk to us? Join our Discord server by clicking the badge below.
 
-[![Discord](https://img.shields.io/badge/-discord-black?style=social&logo=discord)](https://discord.gg/FQtyMWFZQ9)
+[![DISCORD - Join](https://img.shields.io/badge/DISCORD-Join-2ea44f?style=for-the-badge&logo=discord&logoColor=white)](https://discord.gg/9NGVjh9h)
```

Comment:
is this discord link expires? previous one doesn't expires at all. need a clarification

Reply:
I don't have idea regarding that one. I have created this link though invite option in the discord.

Reply:
then please keep the older link.